src/v2/Apps/Categories/Components/GeneFamily.tsx

```diff
@@ -0,0 +1,64 @@
+import { Box, Link, Spacer, Text, themeProps } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { GeneFamily_geneFamily } from "v2/__generated__/GeneFamily_geneFamily.graphql"
+import { sortBy } from "lodash"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+interface GeneFamilyProps {
+  geneFamily: GeneFamily_geneFamily
+}
+
+type Genes = GeneFamily_geneFamily["genes"]
+
+const alphabetizeGenes = (genes: Genes): Genes =>
+  sortBy(genes, gene => gene?.displayName || gene?.name)
+
+export const GeneFamily: React.FC<GeneFamilyProps> = props => {
+  const { geneFamily } = props
+  const { name, genes } = geneFamily
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
```

Comment:
@damassi is this SSR-safe?

Reply:
Unfortunately not; we'll want to use our fresnel `<Media at='xs'>` component to get at this. Or....

Reply:
Could just use a styled-component with a media query.

Reply:
See https://github.com/artsy/force/pull/8379#discussion_r713344320